flymake-shellcheck.el

```diff
@@ -30,13 +31,22 @@
 
 ;;; Code:
 
+(defgroup flymake-shellcheck nil
+  "Shellcheck backend for Flymake."
+  :prefix "flymake-shelcheck-"
```

Comment:
```suggestion
  :prefix "flymake-shellcheck-"
```

Reply:
Fixed in f4892aa087ec7a6ad00584b2a415626adbfee6d3.